_pages/homepage.md

```diff
@@ -53,93 +53,144 @@ flow:
+            This includes a minimal (nano) headless system Android (64bit) profile suitable for use with the FVP. Full Android boot will be supported on the Morello hardware platform
+
+            Support for Pure capability (Pure-cap) applications (along with several example ports) is provided by a Morello ACK (Android Common Kernel) and Bionic library variants built using the CHERI LLVM/Clang toolchain.
+
+            For more information on the status of the Android environment, please refer to `android-readme.rst` in the Documentation repository
+
+            Further information on the timeline for ongoing work in the Morello Kernel and the use of libshim in C libraries is available below.
+
+            ## Linux Enablement (2022 onwards)
+
+            Support for a Morello Linux environment is still under development. 
+
+            ### Initial Proof of Concept Linux environment: Expected end of 2021 (on FVP)
+            * Based around CHERI LLVM/Clang toolchain & Morello ACK.  Provides a prototype (shimmed) musl libC port and hello world userspace example integrated into a standard (A64) rootfs. 
+            * This provides a starting point for incremental enhancements targeting both the Morello hardware and FVP platforms.
+
+            ### Examples of planned work packages, 2022 onwards:
+
+            * Initial Linux environment support for hardware platform (and support for native compilation)
+            * Further evolution of musl lib C support & Pure-cap userspace application ports
+            * Staged introduction of Morello Kernel PCuABI features
+            * Introduction of a Morello GCC/GlibC based environment
+            * Proof of concept work to demonstrate a Morello aware Distro framework
+
+            Further information on the timeline for ongoing work in the Morello Kernel and the use of libshim in C libraries is available below.
+
+            ## CheriBSD
+
+            A mature [CheriBSD](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-morello-software.html) environment for Morello is implemented and hosted separately by the University of Cambridge Computing Laboratory. This provides a memory-safe kernel and userspace, along with examples ports of application frameworks.
 
-            This includes initial modifications to a standard (arm64-v8a) AOSP to support capabilities.
-            Including a modified Bionic library and some pure-capability application ports.
       - format: text
         style: bg-white p-4 my-3
         text_content:
           text: |-
-            ## CheriBSD
+            ## Component support & tooling
+
+            ### Firmware
+            Platform ports of standard open source firmware components: SCP firmware, Trusted Firmware TF-A, UEFI EDK II are available for the Morello platform. These are aarch64 platform ports, with some additional low level hardware initialization to enable support of Capabilities in higher level software.
```

Comment:
AArch64 not aarch64, and Capabilities should not be capitalised

Reply:
Ack. Happy to run with AArch64 & capabilities throughout (capitalising only if it starts the sentence).